packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -73,6 +109,8 @@ class AppSizeController {
   static const identicalFilesError =
       'Failed to load diff: OLD and NEW files are identical.';
 
+  static const deferredRootName = 'ArtificialRoot';
```

Comment:
This should no longer be necessary with the changes in https://github.com/flutter/devtools/pull/4346